lib/rubocop/cop/lint/ineffective_access_modifier.rb

```diff
@@ -96,9 +96,7 @@ def format_message(modifier)
         end
 
         # rubocop:disable Metrics/CyclomaticComplexity
-        def ineffective_modifier(node, modifier = nil, &block)
-          ignored_methods = nil
-
+        def ineffective_modifier(node, ignored_methods = nil, modifier = nil, &block)
```

Comment:
Unrelated to the fix - we should switch to keyword params here.

Reply:
Sure. However, if changed to keyword arguments, it should be prevent new `Layout/LineLength` or `Metrics/MethodLength`'s offense. This method is limited, so I'd like to keep the changes to a minimum at this time.